data/mods/Xedra_Evolved/mapgen/cave.json

```diff
@@ -68,5 +68,50 @@
     "method": "json",
     "nested_mapgen_id": "goblin_spider_cocoon",
     "object": { "mapgensize": [ 1, 1 ], "place_item": [ { "item": "corpse", "x": 0, "y": 0 } ] }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "cave_rat" ],
+    "object": {
+      "fill_ter": "t_rock_floor",
+      "rotation": [ 0, 3 ],
+      "rows": [
+        "........................",
```

Comment:
There doesn't appear to be a rat king in your rat king cave

Reply:
Ah yeah, it would appear someone copied over the wrong json. Give me a second